lib/id3c/cli/command/etl/fhir.py

```diff
@@ -180,7 +180,13 @@ def process_diagnostic_report_bundle_entry(db: DatabaseSession, bundle: Bundle,
     LOG.debug(f"Processing DiagnosticReport Resource «{entry.fullUrl}».")
 
     for reference in resource.specimen:
-        if not matching_system(reference.identifier, INTERNAL_SYSTEM):
+        if not reference.identifier:
+            specimen = reference.resolved(Specimen)
+            reference.identifier = specimen.identifier[0]
```

Comment:
It'd be nice to find all the `specimen.identifier` items which match our system, instead of only the first. This would correctly handle Specimens with multiple identifiers where ours isn't first.

Reply:
Yes definitely!